config.yaml

```diff
@@ -38,198 +38,106 @@ docusaurus:
     course_name: SO
     sidebar: js
     structure:
-      - Introduction:
-        - Intro:
-            name: README
-            location: content/landing-page
+      - Introduction: content/landing-page/README.md
       - Lecture:
-        - Software Stack:
-            name: Software-Stack
-            location: /build/embed_reveal/Software-Stack
-        - Data:
-            name: Data
-            location: /build/embed_reveal/Data
-        - Compute:
-            name: Compute
-            location: /build/embed_reveal/Compute
-        - IO:
-            name: IO
-            location: /build/embed_reveal/IO
-        - Application Interaction:
-            name: Application-Interaction
-            location: /build/embed_reveal/Application-Interaction
+          path: /build/embed_reveal
+          subsections:
+            - Software Stack: Software-Stack/Software-Stack.mdx
+            - Data: Data/Data.mdx
+            - Compute: Compute/Compute.mdx
+            - IO: IO/IO.mdx
+            - Application Interaction: Application-Interaction/Application-Interaction.mdx
```

Comment:
Does the builder create the `.mdx` files? The [makefile](https://github.com/open-education-hub/operating-systems/blob/master/content/common/makefile/slides.mk) simply creates a `.md` file.

Reply:
[Yes](https://github.com/open-education-hub/operating-systems/blob/bf54209d3c09803b914a0ddf517929ac8d7f9ad2/config.yaml#L27), those are from the output of the `reveal_embed` plugin.